gsdk/tests/rpc.rs

```diff
@@ -266,11 +274,16 @@ async fn test_runtime_wasm_blob_version_history() -> Result<()> {
             .runtime_wasm_blob_version(Some(no_method_block_hash))
             .await;
 
-        assert!(wasm_blob_version_result.is_err());
-        let error_msg = wasm_blob_version_result.unwrap_err().to_string();
-        assert!(error_msg.starts_with(
-            "Rpc error: RPC error: RPC call failed: ErrorObject { code: MethodNotFound"
+        let err = CallError::Custom(ErrorObject::owned(
+            9000,
+            "Unable to find WASM blob version in WASM blob",
+            None::<String>,
         ));
+        assert_matches!(
+            wasm_blob_version_result,
+            Err(Error::Subxt(SubxtError::Rpc(RpcError::ClientError(e)))) if e.to_string() == err.to_string(),
+            "Error does not match: {wasm_blob_version_result:?}",
+        );
```

Comment:
**Suggestion:** Consider using the `matches!` macro instead of `assert_matches!` for better readability and simplicity.
```suggestion
        assert!(matches!(
            wasm_blob_version_result,
            Err(Error::Subxt(SubxtError::Rpc(RpcError::ClientError(e)))) if e.to_string() == err.to_string()
        ), "Error does not match: {wasm_blob_version_result:?}");
```